scripts/jest/setupTests.js

```diff
@@ -296,14 +296,19 @@ if (process.env.REACT_CLASS_EQUIVALENCE_TEST) {
 
 // We mock createHook so that we can automatically clean it up.
 let installedHook = null;
+let outgoingHook = null;
 jest.mock('async_hooks', () => {
   const actual = jest.requireActual('async_hooks');
   return {
     ...actual,
     createHook(config) {
-      if (installedHook) {
-        installedHook.disable();
+      // We unmount when there's more than two hooks installed.
+      // We use two because the build of server.node actually installs two hooks.
+      // One in each build.
```

Comment:
This reveals that we're actually installing two `async_hooks` when you import `server.node`. One in the Node build and one in the Node Web Streams build. 😬 Since it's a side-effect of requiring these.

Basically you have to deep link to make it optimized.